### PR TITLE
1730 update wqp colors

### PR DESCRIPTION
**Pull request by lilyhoutman**
https://internal.cida.usgs.gov/jira/browse/WQP-1730

Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Review by mbucknell — APPROVED** [review]
I would again encourage the use of USWDS colors. This makes it simpler to guarentee that we meet accessibility requirements. USWDS colors can be specified by using mixins. See https://designsystem.digital.gov/design-tokens/color/overview/

**Comment by lilyhoutman** [comment]
> I would again encourage the use of USWDS colors. This makes it simpler to guarentee that we meet accessibility requirements. USWDS colors can be specified by using mixins. See https://designsystem.digital.gov/design-tokens/color/overview/

I definitely agree. I think I've been getting some conflicting tasks on my JIRA tickets (which isn't your fault). Not 100% sure where all of the ideas are coming from, but I've been told to change things like font and colors, which goes against USWDS standards. I'm going to update this branch with USWDS colors.

**Comment by mbucknell** [comment]
@lilyhoutman , you can likely find USWDS colors that are close to the requested. The advantage of course is that when you need contrasting colors (like for text on the background), you can choose an appropriate USWDS color and be guaranteed that it meets WCAG guidelines.

File: assets/scss/custom/custom.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@ $black: #000;
 @import "../../node_modules/uswds/dist/scss/uswds";
 
 body {
-  background-color: #BDCCD4;
+  background-color: #bbc8f5; //USWDS indigo-cool-20
```
Comment (mbucknell) [inline assets/scss/custom/custom.scss:13]: You should be able to use the mixin like this @include u-bg('indigo-cool-20'). Here is an example in one of my other projects where we use the mixins: https://github.com/usgs/waterdataui/blob/master/assets/src/styles/main.scss#L58

**Pull request merged**